selene/src/opts.rs

```diff
@@ -7,9 +7,9 @@ use structopt::{clap::arg_enum, StructOpt};
 #[structopt(setting(structopt::clap::AppSettings::ArgsNegateSubcommands))]
 #[structopt(setting(structopt::clap::AppSettings::SubcommandsNegateReqs))]
 pub struct Options {
-    /// A glob to match files with to check
-    #[structopt(long, default_value = "**/*.lua")]
-    pub pattern: String,
+    /// Comma-separated globs to match files to check
+    #[structopt(long, default_value = "**/*.lua,**/*.luau")]

```

Comment:
We probably shouldn't default match `.luau` unless the "roblox" feature flag is enabled (which will eventually be renamed to Luau).

You can do this with something like:

```suggestion
    #[cfg_attr(feature = "roblox", structopt(long, default_value = "**/*.lua,**/*.luau"))]
    #[cfg_attr(not(feature = "roblox"), structopt(long, default_value = "**/*.lua,**/*.luau"))]
```